hieraserver/rest/rest.go

```diff
@@ -70,7 +81,25 @@ func startServer(cmd *cobra.Command, _ []string) {
 	hiera.DoWithParent(context.Background(), provider.MuxLookupKey, configOptions, func(ctx px.Context) {
 		ctx.Set(`logLevel`, px.LogLevelFromString(logLevel))
 		router := CreateRouter(ctx)
-		err := http.ListenAndServe(addr+":"+strconv.Itoa(port), router)
+
+		server := &http.Server{
+			Addr:    addr + ":" + strconv.Itoa(port),
+			Handler: router,
+		}
+
+		var err error
+		var tlsConfig *tls.Config
+		tlsConfig, err = makeTLSconfig()
```

Comment:
Please skip preceding `var` declarations and use `:=`.

Reply:
done